### DISCUSSION TITLE
React Particles not working when trying to create a reusable component

### DISCUSSION THREAD

**Discussion by terry9737** [matteobruni/tsparticles#2378] [Q&A] [answered]
Hi,
I am trying to apply a Particles background on every card I have on a landing page, and the particles component seem to work only on 1 card. When I apply the Particles component on each of my 4 cards only 1 is showing the desired effect. It seems like the component works only per 1 element. Any solutions for fixing this? 
Any help will be much appreciated!

**Comment by terry9737** [comment, marked as answer]
edit: it seems like I found a fix to this problem by simply assigning an id for each component :)

**Reply by WilliamBotbyl** [reply]
how do I initialize it if I want the stars instead of the bubbles? been trying to figure it out and I can't, sorry im kind of new to coding

**Reply by matteobruni** [reply]
Checkout this configuration: https://codepen.io/matteobruni/pen/NWajNRb

**Reply by WilliamBotbyl** [reply]
I did but im using react so im trying to set it to the starry night one, sorry for all the questions

**Reply by matteobruni** [reply]
The configuration object is the same everywhere, you can start copying one, and adjusting it to your needs.

I can’t help you that much without some code shown.
